lib/qesapdeployment.pm

```diff
@@ -356,7 +208,9 @@ sub qesap_execute {
 
     my $verbose = $args{verbose} ? "--verbose" : "";
     my %paths = qesap_get_file_paths();
-    my $exec_log = "/tmp/qesap_exec_$args{cmd}_$args{cmd_options}.log.txt";
+    my $log_postfix = $args{cmd_options} ? "_$args{cmd_options}" : "";
+    my $exec_log = "/tmp/qesap_exec_$args{cmd}$log_postfix.log.txt";
```

Comment:
Don't see the need for `$log_postfix` variable. You can accomplish the same by:
```
my $exec_log = $args{cmd_options} ? "/tmp/qesap_exec_$args{cmd}_$args{cmd_options}.txt" : "/tmp/qesap_exec_$args{cmd}.txt";
```
If you feel that line is too long and repeating too much text, you could also do:
```
my $exec_log = "/tmp/qesap_exec_$args{cmd}";
$exec_log .= "_$args{cmd_options}" if ($args{cmd_options});
$exec_log .= '.txt';
```